packages/node/test/setup/init/set-env-vars.ts

```diff
@@ -1,4 +1,3 @@
-process.env.MASTER_KEY_MNEMONIC = 'achieve climb couple wait accident symbol spy blouse reduce foil echo label';
```

Comment:
Can the rest of the env variables here be removed too or not yet?

Reply:
Not yet, I'm still working on moving the rest of the secrets. I'll clean it in the process.